politeiawww/cmd/politeiawwwcli/commands/help.go

```diff
@@ -0,0 +1,19 @@
+package commands
+
+import "fmt"
+
+type HelpCmd struct {
+	Args struct {
+		Topic string `positional-arg-name:"topic" description:"get information about available commands"`
+	} `positional-args:"true"  required:"true"`
+}
+
+func (cmd *HelpCmd) Execute(args []string) error {
+	topic := cmd.Args.Topic
```

Comment:
Small nitpick.  Don't create variables if you don't use them more than once and/or go over 80 columns.